samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -166,6 +166,7 @@ tests:
     integration_platforms:
       - nrf52840dk/nrf52840
   sample.mcumgr.smp_svr.ram_load:
+    sysbuild: true
```

Comment:
It's already defined in the `common ` section, no need to duplicate it.
```
common:
  sysbuild: true
```

Reply:
Good catch- dropped this